omnibus/package-scripts/installer-deb/postinst

```diff
@@ -36,53 +38,26 @@ set -e
 case "$1" in
     configure)
         add_user_and_group 'dd-installer' $PACKAGES_DIR
-        add_user_and_group 'dd-agent' $PACKAGES_DIR/datadog-agent
-        usermod -aG dd-agent dd-installer
```

Comment:
not being in the agent group will prevent writing logs and touch other dirs owned by agent

Reply:
Actually since the installer is run as root and changes it's groups manually, including dd-agent it should work

Reply:
It does from naive testing, I'll double check on a brand new VM